java-spiffe-core/src/main/java/io/spiffe/internal/JwtSignatureAlgorithm.java

```diff
@@ -101,14 +97,14 @@ public static Family parse(final String s) {
             } else if (s.equals(EC.getName())) {
                 family = EC;
             } else {
-                family = OTHER;
+                throw new IllegalArgumentException("Unsupported JWT token family algorithm: " + s);
```

Comment:
nit: `JWT token` is a little redundant, I think just removing `token` from the message should be fine
```suggestion
                throw new IllegalArgumentException("Unsupported JWT family algorithm: " + s);
```